app/controllers/greetings_controller.rb

```diff
@@ -1,9 +1,8 @@
 class GreetingsController < ApplicationController
-  before_action :set_greeting, only: %i[ show edit update destroy ]
+  before_action :set_greeting, only: %i[show edit update destroy]
 
   # GET /greetings or /greetings.json
   def index
     @greetings = Greeting.all
   end
-
 end
```

Comment:
- [ ] For some reason am not able to see greetings being displayed on the screen check if you set up well your cors file or if is something else

Reply:
Hi, thank you.. it's important that you run the Rails App on port 3001.
The CORS is correctly setup.

```
Rails.application.config.middleware.insert_before 0, Rack::Cors do
  allow do
    origins 'http://127.0.0.1:3000'

    resource '*',
      headers: :any,
      methods: [:get]
  end
end
```

Reply:
Did you also run `rails db:seed`?